### DISCUSSION TITLE
Move individual commands to parameters

### DISCUSSION THREAD

**Discussion by savchenko** [linrongbin16/fzfx.nvim#525] [Ideas]
Currently `fzfx.nvim` creates more than 30 (!) individual commands:

![image](https://github.com/linrongbin16/fzfx.nvim/assets/300146/af04d598-0237-45aa-93fb-99980462693e)

How about moving `M/W/P/R` options into parameters? At minimum, this would save screen space and reduce confusion amongst new adopters.

**Comment by linrongbin16** [comment]
hi @savchenko, the idea looks pretty good to me, I didn't think in that way!

I will look into it.

**Reply by linrongbin16** [reply]
This is already done long ago.
